scripts/rf_accounts.py

```diff
@@ -42,8 +51,38 @@
     logger.info( "rf_accounts Trace" )
 
 # Set up the Redfish object
-redfish_obj = redfish.redfish_client( base_url = args.rhost, username = args.user, password = args.password )
-redfish_obj.login( auth = "session" )
+try:
+    redfish_obj = redfish.redfish_client( base_url = args.rhost, username = args.user, password = args.password , timeout=5, max_retry=3)
+    redfish_obj.login( auth = "session" )
+except RedfishPasswordChangeRequiredError as e:
```

Comment:
When we get into this exception case, I'd like to recommend the following logic:

```
except RedfishPasswordChangeRequiredError as e:
    if args.setpassword is not None and args.user == args.setpassword[0]"
        redfish_utilities.set_password_via_uri( redfish_obj, <uri from exception>, args.setpassword[1]
    else:
        print( "Password change required; run rf_accounts.py with the --setpassword to change your account's password" )
```

Since the exception contains the exact URI to modify, that will prevent us from having to dig through the service to find the right user account to change.
```